### PR TITLE
Correct: Test app name to be used by float command

### PR DESCRIPTION
**Pull request by andygout**
cc @ironsidevsquincy @leggsimon 

Multi-region app `ft-next-api` has been added to Heroku. Not sure how to currently test that it is 'working fine' (as per step 5 in [Setting up Multi Region Apps](http://financial-times.github.io/next/docs/developer-guide/multi-region-apps/)), though (other than seeing it listed on Heroku).

Have verified that `float` command works under `TEST: $ make build-production test provision` in this branch's [build](https://circleci.com/gh/Financial-Times/next-api/17?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link).

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 include n.Makefile
 
-TEST_APP := "ft-next-graphql-api-${CIRCLE_BUILD_NUM}"
+TEST_APP := "ft-next-api-${CIRCLE_BUILD_NUM}"
```
Comment (ironsidevsquincy) [inline Makefile]: `...ft-next-api-branch-...`

Comment (andygout) [inline Makefile]: How come?

Comment (ironsidevsquincy) [inline Makefile]: That's the format everything else uses

https://github.com/Financial-Times/next-front-page/blob/master/Makefile#L3
https://github.com/Financial-Times/next-stream-page/blob/master/Makefile#L3
https://github.com/Financial-Times/next-article/blob/master/Makefile#L3

etc

Comment (andygout) [inline Makefile]: Yup - was taking my lead from [`next-graphql-api`](https://github.com/Financial-Times/next-graphql-api/blob/master/Makefile#L3).

**Pull request merged**